content/pages/sphinx.html

```diff
@@ -0,0 +1,243 @@
+    {% endmarkdown %}
+    {% endcall %}
+
+    {% call marketing.highlight_item(
+        icon="fa-image-landscape",
+        header="Multiple docs versions",
+        color="violet") %}
+    {% markdown %}
+    Host the docs for all the supported versions of your software,
+    so that users can access the exact one they need.
+    Works great for different development workflows,
+    as well as public documentation.
+    {% endmarkdown %}
+        {% endcall %}
+    {% endcall %}
+</section>
+
+<div class="ui very padded container">
+
+{{ basic.header_icon(
+  header="Some of our users",
+  subheader="Thousands of businesses and open source projects use Read the Docs daily",
+  icon="purple fa-message-lines",
+  icon_style="--fa-secondary-color: blueviolet;"
+) }}
+
+{#
+  Cards for an articles or case studies view
+
+  See https://fomantic-ui.com/views/card.html
+#}
+<div class="ui three link stackable cards">
+    <a href="https://flask.palletsprojects.com/en/2.2.x/" class="card">
+    <div class="ui small centered image">
+        <img src="https://upload.wikimedia.org/wikipedia/commons/3/3c/Flask_logo.svg" alt="A random image">
+      </div>
+      <div class="content">
+        <div class="header">Flask</div>
+        <div class="meta">
+          Case study
+        </div>
+        <div class="description">
+            Flask is a Python web framework built with a small core and easy-to-extend philosophy.
+        </div>
+      </div>
+    </a>
+
+    <a href="https://docs.jupyter.org/en/latest/" class="card">
+    <div class="ui medium centered image">
+        <img src="https://cdn.icon-icons.com/icons2/2699/PNG/512/jupyter_logo_icon_169453.png" alt="A random image">
+      </div>
+      <div class="content">
+        <div class="header">Jupyter</div>
+        <div class="meta">
+          Case study
+        </div>
+        <div class="description">
+            Jupyter Notebook is a web-based interactive computational environment for creating notebook documents.
+        </div>
+      </div>
+    </a>
+
+    <a href="https://docs.ceph.com/" class="card">
+    <div class="ui small centered image">
+        <img src="https://ceph.io/assets/bitmaps/Ceph_Logo_Standard_RGB_Black_120411_fa.png" alt="A random image">
+      </div>
+      <div class="content">
+        <div class="header">Ceph</div>
+        <div class="meta">
+          Case study
+        </div>
+        <div class="description">
+            Ceph uniquely delivers object, block, and file storage in one unified system.
+        </div>
+      </div>
+    </a>
+</div>
+
+<div class="ui very padded basic vertical segment">
+  <div class="ui horizontal divider">Proven across time</div>
+</div>
+
+{# Grid view stuff #}
+<div class="ui stackable equal width grid">
+  <div class="stretched row">
+    <div class="column">
+      <div class="ui very padded raised large segment">
+        <h4 class="ui header large">
+          1 million served. Every day.
+        </h4>
+
```

Comment:
I'd like to see a small paragraph on this card because it looks too empty otherwise. I like the title and these numbers, but I feel a small paragraph would be 💯